### DISCUSSION TITLE
Google Summer of Code 2024

### DISCUSSION THREAD

**Discussion by evenstensberg** [webpack/webpack#17441] [General]
Room for webpack in GSoC 2024. Here you can ask questions to our organization members that are involved in GSoC. 

Mentors: @snitin315, @dhruvdutt and @anshumanv 
Organization Admins: @evenstensberg

**Comment by KiroSenpai3** [comment]
is webpack going to participate in gsoc 2024? 
if Yes, i want to start my open source contribution in webpack. where do I get started with?

**Comment by evenstensberg** [comment]
@KiroSenpai3 Yes. Here's a few resources:

- https://even.dev/2023/08/18/contributing-to-webpack/
- https://medium.com/@evenstensberg/writing-a-solid-google-summer-of-code-proposal-a200fc6e785b

**Comment by aialok** [comment]
Hey @evenstensberg! Can you please tell me how to get started with webpack like I do have good knowledge of javascript but I have never work in big open source project.  I do read your blog but I still dont know what are the things more I need to learn in order to contribute to the webpack ?
If you tell me from where and how I get started with webpack to someone beginner like me. It would be great : )
Thank you !
Also If there is any slack/Discord Channel where I can ask question it would be really great.

**Reply by evenstensberg** [reply]
You should at least learn how to set up webpack as a web dev project.

**Reply by aialok** [reply]
Thank You @evenstensberg ! is there any slack channel to for support and doubts ?

**Reply by evenstensberg** [reply]
Send me a message on Twitter

**Comment by kevinking1983** [comment]
@evenstensberg is there any chance to contribute now as I have descent knowledge in Nodejs

**Reply by evenstensberg** [reply]
Sure. Best way is to scan for "help wanted" labeled issues in any of webpack/webpack contrib repos.

**Reply by kevinking1983** [reply]
Hi @evenstensberg  can I know the projects ideas and the skills required for them

**Reply by evenstensberg** [reply]
Ideas list will be published after organisations are public at the gsoc site

**Reply by kevinking1983** [reply]
Hi sir, I have mailed you regarding my interest in contributing to GSoC 2024

**Comment by kolawolecreates** [comment]
Hello @evenstensberg 

I am Kolawole and I have picked interest in web pack for the google summer of code. I have a project I am leaning towards but I really don't know how to proceed as I can't find a forum or anything.

**Comment by nyngwang** [comment]


**Reply by Lovelyfin00** [reply]
Hi everyone
I'm Loveth, a GSOC applicant and would like some help with the link to the discord channel
Thank you

**Comment by maverox** [comment]
> To @evenstensberg:
> 
> Hi, I'm a regular user of webpack and I care about its current status very much. In short, I have noticed some bad trends happened recently due to the upcoming Google Summer of Code 2024. Quoted my main point of suggestion:
> 
> > I think it would be great to open a thread for every issue/problem we need to solve here in Discord so we can avoid Q&A style comments in the current repo [...]
> 
> For more details, please see my [comments](https://discord.com/channels/1180618526436888586/1180618528131399843/1210327479747739688) in your Discord - General channel.
> 
> **update**: This is still happening. (New at top)
> 
> <img alt="" width="500" src="https://private-user-images.githubusercontent.com/24765272/307567350-63fdfa9d-a351-42dd-8086-52026ad4140a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwODM2MDMsIm5iZiI6MTcxMDA4MzMwMywicGF0aCI6Ii8yNDc2NTI3Mi8zMDc1NjczNTAtNjNmZGZhOWQtYTM1MS00MmRkLTgwODYtNTIwMjZhZDQxNDBhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDE1MDgyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlOTljYjgzYjVhOGMzYmZjYjdmMDlkZjViZDkyNmMwZDdmNTg5ZTUxMjU3YWI3NzU3MWQ0NTI2YTYxNzUxYTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ri2USwPxozwFIz-eK94pS3jSqjjWaf50DM3fLbD1JbI"> <img alt="" width="500" src="https://private-user-images.githubusercontent.com/24765272/307503146-8916e0f2-7673-4455-9efc-af600abda019.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwODM2MDMsIm5iZiI6MTcxMDA4MzMwMywicGF0aCI6Ii8yNDc2NTI3Mi8zMDc1MDMxNDYtODkxNmUwZjItNzY3My00NDU1LTllZmMtYWY2MDBhYmRhMDE5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDE1MDgyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI1ZWFiZDkwN2UzNjE3MzkyNThlYjFiZjM0NjM3ODY1ZjAzYzIyOGM1ZGM3YjM0Nzg1NzY5YWEzMWMwYjhkYmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bCR7AM1WXdkti5ulgSS8K1RkAv7mE2hfOeGmPlawbeU">

That is such a nice idea, as it would ensure comment of issues don't get polluted by random Questions.

**Reply by evenstensberg** [reply]
We have a gsoc channel on Discord but most of people ignore that and post in general or randomly here at GitHub

**Comment by Lovelyfin00** [comment]
Hii everyone
Sorry for posting this here
But I’ve been searching for invite to the discord channel for a couple of weeks now. I even joined the gitter channel and requested for the link but no response till now.

I would highly appreciate it if someone can send me the link or an invitation. Thank you

**Comment by maverox** [comment]
https://discord.com/invite/4wkMtuHt

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by Lovelyfin00** [reply]
Thanks a lot 🙏

**Comment by junaidzeb123** [comment]
Is web pack coming in gsoc 2025 if yes please tell me how i can start contributing now?

**Comment by evenstensberg** [comment]
https://github.com/webpack/webpack/discussions/19114
